angular-ngrx-scss/src/app/shared/components/repo-controls/repo-controls.component.html

```diff
@@ -52,12 +53,12 @@
           {{ (selectSortFilter$ | async)?.split('_')?.join(' ') | lowercase }}
         </span>
       </p>
-      <span class="results__clear-container">
-        <div class="close-background">
+      <button type="button" class="results__clear" (click)="handleClearClick()">
```

Comment:
[nitpicking]. do we need the type button here? l think that is the default value? Except when setting explicitly to "submit" for example?

Reply:
@asincole, I'm pretty sure the default type is "submit" 🙂

Reply:
Yes indeed, according to w3c the default value for most of the browsers is "Submit". The only browser where the default type is different is in IE.